consensus/consensus.go

```diff
@@ -195,9 +200,11 @@ type StakeInfoFinder interface {
 
 // New creates a new Consensus object
 // TODO: put shardId into chain reader's chain config
-func New(host p2p.Host, ShardID uint32, leader p2p.Peer, blsPriKey *bls.SecretKey) (*Consensus, error) {
+func New(host p2p.Host, ShardID uint32, leader p2p.Peer, blsPriKey *bls.SecretKey, isHarmonyAccount bool, delayCommit int) (*Consensus, error) {
```

Comment:
Since a non-zero `delayCommit` is a special case, please consider not giving it as a mandatory argument, but introduce `SetCommitDelay` instead as a method that can be optionally called if delay behavior is needed.